introduction.md

```diff
@@ -0,0 +1,2 @@
+Hi my name is Daniel, now I start working on the mastering github.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.